RecoEgamma/PhotonIdentification/plugins/PhotonIDValueMapProducer.cc

```diff
@@ -504,10 +556,17 @@ ::computeWorstPFChargedIsolation(const T& photon, const U& pfCandidates,
 
       if (iCand->pt() < ptMin)
 	continue;
+      
+      float dxy=-999, dz=-999;
+      getImpactParameters(iCand, isAOD, pv, dxy, dz);
 
-      auto dxydz = getTrackDxyDz(iCand, vtx->position(), isAOD);
-      if ( fabs(dxydz.first) > dxyMax) continue;
-      if ( fabs(dxydz.second) > dzMax) continue;
+
+      //      const reco::Track *theTrack = getTrackPointer( iCand, isAOD );
+      //float dxy = theTrack->dxy(vtx->position());
+      if( fabs(dxy) > dxyMax) continue;
+      //
+      //float dz = theTrack->dz(vtx->position());
```

Comment:
is this commented out code needed?
Please remove or add comments inline in the code why the commented out block is relevant

Reply:
Changes implemented here:
https://github.com/lsoffi/cmssw/blob/CMSSW_9_4_0_pre3_TnP/RecoEgamma/PhotonIdentification/plugins/PhotonIDValueMapProducer.cc